File: StraviaTECApi/Models/Actividad.cs
using System;
using System.Collections.Generic;

namespace StraviaTECApi.Models
{
    public partial class Actividad
    {
        public string Usuariodeportista { get; set; }
        public DateTime Fechahora { get; set; }
        public TimeSpan? Duracion { get; set; }
        public double? Kilometraje { get; set; }
        public string Tipoactividad { get; set; }
        public string Recorridogpx { get; set; }
        public string Nombreretocarrera { get; set; }
        public string Adminretocarrera { get; set; }
        public int Banderilla { get; set; }

        public virtual Deportista UsuariodeportistaNavigation { get; set; }
    }
}

File: StraviaTEC/src/app/modelos/actividad.ts
export class Actividad {
    constructor(
        public UsuarioDeportista: string,
        public Nombre: string,
        public FechaHora: string,
        public Duracion: string,
        public Kilometraje: number,
        public TipoActividad: string,
        public RecorridoGPX: string,
        public NombreRetoCarrera: string,
        public AdminRetoCarrera: string,
        public Banderilla: number
    ){}
}
